src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2547,9 +2545,19 @@ namespace winrt::TerminalApp::implementation
         _RemoveTabViewItem(tabViewItem);
     }
 
-    TermControl TerminalPage::_InitControl(const TerminalSettings& settings, const ITerminalConnection& connection)
+    TermControl TerminalPage::_InitControl(const TerminalSettingsStruct& settings, const ITerminalConnection& connection)
     {
-        return TermControl{ TerminalSettings::CreateWithParent(settings), connection };
+        // Give term control a child of the settings so that any overrides go in the child
+        // This way, when we do a settings reload we just update the parent and the overrides remain
+        const auto child = TerminalSettings::CreateWithParent(settings);
+        TermControl term{ child.DefaultSettings(), connection };
+
+        if (child.UnfocusedSettings())
+        {
+            term.UnfocusedAppearance(child.UnfocusedSettings());
+        }
```

Comment:
since it is safe to set UnfocusedAppearance to null, simplify.

```suggestion
        term.UnfocusedAppearance(child.UnfocusedSettings() /* null is okay */);
```